flink-table/flink-table-dist/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.apache.flink</groupId>
+		<artifactId>flink-table</artifactId>
+		<version>1.8-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+
+	<artifactId>flink-table-dist_${scala.binary.version}</artifactId>
```

Comment:
i would prefer the `uber` suffix vs `dist`; it's what we've been using so far

Reply:
This name was a suggestion from @aljoscha, what do you think? I don't have a strong opinion here.

Reply:
It's to keep it in line with `flink-dist`. What do you think?

Reply:
I'd reserve the `dist` suffix for self-contained distribution, such as flink-dist.
table-dist is not self-contained; you can't run anything with it on it's own. As far as I'm concerned it only bundles dependencies, similar to what `flink-shaded-hadoop2-uber` does.

Note that I would propose to name the _jar_  `flink-table`, without any suffix. Then we wouldn't break any user-scripts that rely on jar names.

Reply:
Could do that as well. 👌

Reply:
Sounds good to me.